e2e/cephfs.go

```diff
@@ -149,14 +165,20 @@ var _ = Describe("cephfs", func() {
 			logsCSIPods("app=csi-cephfsplugin", c)
 		}
 		deleteConfigMap(cephfsDirPath)
```

Comment:
@Madhu-1 better to have a error check for this configmap deletion too? or we will delete the configmap too with "deleteResource" generic function to be consistent with other resource deletion?